main.py

```diff
@@ -0,0 +1,74 @@
+from datetime import date
+
+from fastapi import FastAPI, HTTPException
+import requests
+from pydantic import BaseModel
+
+import models
+from database import engine, SessionLocal
+
+app = FastAPI()
+models.Base.metadata.create_all(bind=engine)
+
+
+class PriceData(BaseModel):
+    symbol: str
+    open: float
+    high: float
+    low: float
+    price: float
+    volume: int
+    latest_trading_day: date
+    previous_close: float
+    change: float
+    change_percentage: float
+
+    class Config:
+        from_attributes = True
+
+
+db = SessionLocal()
+
+
+@app.get("/stock/quote")
+def get_stock_quote():
+    url = "https://alpha-vantage.p.rapidapi.com/query"
+
+    querystring = {"function": "GLOBAL_QUOTE", "symbol": "MSFT", "datatype": "json"}
```

Comment:
query string values should be passed to the endpoint by the caller i.e. Frontend, POSTMAN, etc.